api/src/main/java/edu/cornell/mannlib/vitro/webapp/rdfservice/adapters/BulkModelCom.java

```diff
@@ -0,0 +1,34 @@
+package edu.cornell.mannlib.vitro.webapp.rdfservice.adapters;
+
+import org.apache.jena.graph.Graph;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.impl.ModelCom;
+
+public class BulkModelCom extends ModelCom {
+
+	public BulkModelCom(Graph graph) {
+		super(graph);
+	}
+
+	@Override
+	public Model remove(Model m) {
+		if (GraphUtils.isBulkGraphMem(graph)) {
+			BulkGraphMem bulkGraphMem = (BulkGraphMem) GraphUtils.unwrapUnionGraphs(graph);
```

Comment:
In the case GraphUtils.unwrapUnionGraphs(graph) is expensive (time consuming) operation, can we avoid its duplicated invocation (one from isBulkGraphMem(graph)) with the following code:
```suggestion
		Graph unwrapGraph = GraphUtils.unwrapUnionGraphs(graph);
		if (unwrapGraph instanceof BulkModelGraph){
```
The same might work for the three other cases in this PR.

Reply:
fixed